src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -16,32 +16,32 @@
 
 namespace Nethermind.Evm;
 
-using static VirtualMachine;
 using Word = Vector256<byte>;
 
-public ref struct EvmStack<TTracing>
-    where TTracing : struct, IIsTracing
+public ref struct EvmStack
```

Comment:
What's the reason to get rid of this? No benefit on omitting the tracer calls?

Reply:
All the generics in generics were getting really really messy on refactoring to delegate* array; will analyse later when all test pass and do a second pass